static/css/devhub/submission.less

```diff
@@ -105,3 +105,28 @@ div.done-next-steps ul,
     background-position: 0 0;
     padding: 0 0 0 24px;
 }
+
+form#submit-source {
+    #source-code-submission-question {
+        margin-bottom: 0px;
+
+        a {
+            font-weight: bold;
+        }
+    }
+    ul#source-code-example {
+        list-style: '\274F\00A0';
```

Comment:
I talked to @MeridelW about this, and potential confusion of non-working checkboxes in combination with a radio button below them. We agreed it's better to use regular bullet points instead.

So, just `list-style: disc;`.

Reply:
Aww, took me ages* to find out how to get the same bullet points as in your doc 🙄
* not really 😛

Reply:
https://github.com/mozilla/addons-server/pull/9086/commits/6f4bfe0ee0f9a7ff3f3526ab7e91628809090d55